docs/src/development/sanitize-db.md

```diff
@@ -0,0 +1,139 @@
+---
+title: Sanitize databases
+description: Remove sensitive information from databases on non-production environments to control access.
+---
+
+When working on a new feature on your website, you want to use a new branch.
+Using a new branch makes sure that you don't risk to break your live, production website.
+
+Creating a branch to Platform.sh copies both the code and the database to that new development branch.
+These code and database changes need to be tested before being merged to the production website.
+Depending on your processes, internal or external teams may interact with the development environment branch.
+
+Databases of live websites often contain personally identifiable information (PII)
+such as full names, mailing addresses, and phone numbers.
+To ensure people reviewing the code changes can't access information they shouldn't, sanitize the database of any PII that it may contain.
+
+## Before you begin
+
+You need:
+
+- A project with a [MySQL database](../configuration/services/mysql/_index.md)
+- The [Platform CLI](/development/cli/_index.md#cli-command-line-interface) installed locally
+
+Alternatively, if you are using Drupal, you will need:
+
+- A project with a [MySQL database](../configuration/services/mysql/_index.md)
+- [Drush](https://www.drush.org/latest/install/) installed
+
+This guide is about sanitizing MySQL databases.
+
+This guide doesn't address:
+
+- Sanitizing NoSQL Databases ([MongoDB](../configuration/services/mongodb/_index.md), ...).
+- Input validation or input sanitization which are both causes for security vulnerabilities.
+
+## Sanitize the database
+
+Make sure that you only sanitize development environments and **never** on the production environment.
+Otherwise you may lose most or even all of the relevant data stored in your database.
+
+First, let's take a [database dump](../configuration/services/mysql/_index.md#exporting-data) of your development environment.
+You won't alter your production data. This is just a safety precaution.
+To get a database dump, run: `platform db:dump -e <name of your development environment>`.
+
+{{< codetabs >}}
+
+---
+title=Manually
+file=none
+highlight=false
+---
+
+Assumptions:
+
+- `user` is the table where all your PII is stored.
+- `user` is a part of the `staging` development database.
+- `staging` is an exact copy of your production database.
+
+1. Connect to the `staging` database with `platform sql -e staging`.
```

Comment:
```suggestion
1. Connect to the `staging` database by running `platform sql -e staging`.
```
Otherwise it slightly looks at first glance like the connection is to the command